packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -164,16 +164,17 @@
 	$: search, updateQuery();
 
 	$: open ? pauseSorting() : resumeSorting();
+	const toKey = (queryOpt) => queryOpt[value]?.toString() + queryOpt[label]?.toString();
 </script>
 
 <slot />
 
 <QueryLoad data={finalQuery} let:loaded>
 	<div slot="skeleton"></div>
-	{#each loaded ?? [] as queryOpt (queryOpt.value?.toString() + queryOpt.label?.toString())}
+	{#each loaded ?? [] as queryOpt (isNaN(toKey(queryOpt)) ? Math.random().toString() : toKey(queryOpt))}
```

Comment:
`isNaN` here will always be `false` since `toKey` returns a string

Reply:
It actually won't if value and label are undefined (which is the case during HMR for some reason)